v-next/hardhat/test/internal/builtin-plugins/network-manager/json-rpc-request-modifiers/helpers.ts

```diff
@@ -0,0 +1,24 @@
+import type { JsonRpcRequest } from "../../../../../src/types/providers.js";
+
+import { assertHardhatInvariant } from "@ignored/hardhat-vnext-errors";
+
+export function createJsonRpcRequest(
```

Comment:
There’s a very similar function in `v-next/hardhat/src/internal/builtin-plugins/network-manager/json-rpc.ts` called `getJsonRpcRequest`. It might make sense to replace this duplicate function with that one to avoid redundancy. What do you think?

Reply:
Done in [this commit](https://github.com/NomicFoundation/hardhat/pull/5810/commits/c16322151cbb6aedbc5b8ff592dd5e36a505eaca)